src/sentry/static/sentry/app/views/settings/organizationApiKeys/organizationApiKeysList.jsx

```diff
@@ -112,7 +112,7 @@ class OrganizationApiKeysList extends React.Component {
                         message={t('Are you sure you want to remove this API key?')}
                         title={t('Remove API Key?')}
                       >
-                        <span className="icon-trash" />
+                        <IconDelete size="xs" css={{position: 'relative', top: '2px'}} />
```

Comment:
Sometimes I also find the CSS tag very useful, but in this case, would it be better to use emotion `styled` as we usually use?

Reply:
I'm planning to make another pass on this component and replace the bootstrap classnames with a styled component button which should also allow this inline CSS to be removed.